boards/arm/stm32l1_disco/stm32l1_disco.dts

```diff
@@ -87,3 +87,7 @@
 		     &spi2_miso_pb14 &spi2_mosi_pb15>;
 	status = "okay";
 };
+
+&rtc {
```

Comment:
maybe use another PR for the disco board

Reply:
I have been thinking about a dts board overlay added to the samples folder, and this could also be added to that. Unless you think the RTC should be enabled by default.